quickstarts/Function_calling_config.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
<p>Idea, want to include an automatic function calling example after this block? Maybe something like:</p><p> </p><p>Text cell:</p><p> </p><p>## Automatic function calling</p><p>You can also enable automatic function calling in the Python SDK.</p><p> </p><p>Code cell:</p><p> </p><p> </p><p><code>chat = model.start_chat(enable_automatic_function_calling=True)</code></p><p><code>response = chat.send_message("Make this place PURPLE!", tool_config=tool_config)</code></p><p> </p><p>And should we include a "Learning more" section at the end? Maybe...</p><p> </p><p>## Learning more</p><p> </p><p>Check out the function calling [quickstart](<a href="https://github.com/google-gemini/cookbook/blob/main/quickstarts/Function_calling.ipynb)" target="_blank">https://github.com/google-gemini/cookbook/blob/main/quickstarts/Function_calling.ipynb)</a> for an introduction to function calling. You can find another fun function calling example [here](<a href="https://github.com/google-gemini/cookbook/blob/main/quickstarts/rest/Function_calling_REST.ipynb)" target="_blank">https://github.com/google-gemini/cookbook/blob/main/quickstarts/rest/Function_calling_REST.ipynb)</a> using <code>curl</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Done - it's very simple as we can't edit `tool_config` between automatic function calls yet but it illustrates the point..